lib/improver/tests/nbhood/nbhood/test_BaseNeighbourhoodProcessing.py

```diff
@@ -227,19 +242,34 @@ def set_up_cube_lat_long(zero_point_indices=((0, 7, 7),), num_time_points=1,
         DimCoord(np.linspace(0.0, float(num_grid_points - 1),
                              num_grid_points),
                  'latitude',
-                 units='degrees'),
+                 units='degrees',
+                 coord_system=ELLIPSOID),
         1
     )
     cube.add_dim_coord(
         DimCoord(np.linspace(0.0, float(num_grid_points - 1),
                              num_grid_points),
                  'longitude',
-                 units='degrees'),
+                 units='degrees',
+                 coord_system=ELLIPSOID),
         2
     )
     return cube
 
 
+class Test_set_up_cube(IrisTest):
+
+    """Test the set_up_cube method in this module"""
+
+    def test_regrid(self):
+        """
+        Test that set_up_cube returns a cube that can be regridded
```

Comment:
maybe you should add "as this proves that the cubes from set_up_cube have coordinate reference systems", otherwise it's not clear what the link is to regridding.

Reply:
Done.